test/contlinear.jl

```diff
@@ -144,8 +144,11 @@ function linear1test(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64))
         @test MOI.cangetattribute(m, MOI.TerminationStatus())
         @test MOI.getattribute(m, MOI.TerminationStatus()) == MOI.Success
 
-        @test MOI.cangetattribute(m, MOI.PrimalStatus())
-        @test MOI.getattribute(m, MOI.PrimalStatus()) == MOI.FeasiblePoint
+        @test MOI.cangetattribute(m, MOI.ResultCount())
+        @test MOI.getattribute(m, MOI.ResultCount()) == 1
```

Comment:
I am not sure, but maybe we could test for `MOI.getattribute(m, MOI.ResultCount()) >= 1` instead of `== 1`.